build_runner/pubspec.yaml

```diff
@@ -55,3 +55,10 @@ dev_dependencies:
   test_process: ^1.0.0
```

Comment:
we also need to change the minimum version for build_runner_core to be the new version with this change